engine/src/test/java/org/hibernate/validator/test/internal/engine/ValidatorFactoryNoELBootstrapTest.java

```diff
@@ -40,7 +40,7 @@ public class ValidatorFactoryNoELBootstrapTest {
 
 	private final String EL_PACKAGE_PREFIX = "jakarta.el";
 
-	private final String EL_IMPL_PACKAGE_PREFIX = "com.sun.el";
+	private final String EL_IMPL_PACKAGE_PREFIX = "org.glassfish.expressly";
```

Comment:
https://github.com/eclipse-ee4j/expressly/commit/fa5ed8c8be9c290b952b25631cc6b62ab3d3dedb

includes a fix with which resources/META-INF/servcies  are now included in the jar and expressly is getting loaded through the service loader instead of falling back to the `com.sun.el`